src/benchmarks/micro/libraries/System.Text.Encodings.Web/Perf.DefaultJavaScriptEncoder.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#if !NETFRAMEWORK
```

Comment:
Would a conditional include at the project file level be better?

Reply:
Done